### ISSUE TITLE
🐛 Bug Report: not possible to place default value

### ISSUE THREAD

**Issue by s3ppo** [appwrite/appwrite#2590]
### 👟 Reproduction steps

if i want to create an integer or long with a default value, the UI tells me that the value is not valid
![unknown](https://user-images.githubusercontent.com/17125438/148990871-c20e0f89-252c-4d4c-81ec-983581c906d0.png)
![unknown1](https://user-images.githubusercontent.com/17125438/148990882-d4e18d74-c670-4ab7-a8b4-cc6a006262bf.png)

### 👍 Expected behavior

default values on integers and doubles should be possible

### 👎 Actual Behavior

default values on integers and doubles are not possible

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Looks like the body for the create integer API is:

```json
{
    "key": "int",
    "required": false,
    "min": null,
    "max": null,
    "default": "1",
    "array": false
}
```

If you manually made an API call with a number as the default instead of string, it works as expected. So, it looks like this is an Admin Console UI bug and how it's parsing the input.

**Comment by eldadfux** [comment]
@TorstenDittmann fix was merged and will be released in the next Appwrite version. Thanks for reporting the issue @s3ppo!

**Comment by Aragur** [comment]
Any updates when this will be merged? Kind of a big issue at the moment.

**Comment by s3ppo** [comment]
> Any updates when this will be merged? Kind of a big issue at the moment.

I think you can make the work around and set the default value with you coding… its just a bug in the UI

**Comment by Aragur** [comment]
@s3ppo Thanks and yes I'm aware of it :)
But why isn't it implemented in the current release? It was fixed on the 17th January.

**Comment by TorstenDittmann** [comment]
We just released 0.13 and this should be fixed 👍🏻 

You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).

Thank you a lot for reporting this 🙏🏻
